rust/worker/chroma_config.yaml

```diff
@@ -41,6 +41,14 @@ query_service:
     blockfile_provider:
         Arrow:
             max_block_size_bytes: 16384
+            block_manager_config:
+                block_cache_config:
+                    lru:
+                        capacity: 1000
```

Comment:
is there any way to modify this in the future to make the cache size limited instead of capacity limited?